docs/book/src/migration/multi-group.md

```diff
@@ -104,3 +103,92 @@ single-group projects).
 
 [multi-group-issue]: https://github.com/kubernetes-sigs/kubebuilder/issues/923 "Kubebuilder Issue #923"
 [cronjob-tutorial]: /cronjob-tutorial/cronjob-tutorial.md "Tutorial: Building CronJob"
+
+<h1>Using Go/v4</h1>
+
+Generally, we use the prefix for the API group as the directory name. We
+can check `api/v1/groupversion_info.go` to find that out:
+
+```go
+// +groupName=batch.tutorial.kubebuilder.io
+package v1
+```
+
+Then, we'll rename move our existing APIs into a new subdirectory, "batch":
+
+```bash
+mv api/* api/batch
+```
+
+
+After moving the APIs to a new directory, the same needs to be applied to the controllers:
+
+```bash
+mkdir internal/controller/batch
+mv internal/controller/* internal/controller/batch/
+```
+
+
+
+Next, we'll need to update all the references to the old package name. 
+For CronJob, that'll be `main.go` and `controllers/batch/cronjob_controller.go`. 
+
+If you've added additional files to your project, you'll need to track down
+imports there as well.
+
+Finally, fix the PROJECT file manually, the command `kubebuilder edit --multigroup=true`
+sets our project to multigroup, but it doesn't fix the path of the existing APIs.
+For each resource need to modify the path and perform no additional modifications.
+
+For instance, for a file:
+                                                      
+```yaml
+# Code generated by tool. DO NOT EDIT.
+# This file is used to track the info used to scaffold your project
+# and allow the plugins properly work.
+# More info: https://book.kubebuilder.io/reference/project-config.html
+domain: tutorial.kubebuilder.io
+layout:
+- go.kubebuilder.io/v4
+multigroup: true
+projectName: test
+repo: tutorial.kubebuilder.io/project
+resources:
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: tutorial.kubebuilder.io
+  group: batch
+  kind: CronJob
+  path: tutorial.kubebuilder.io/project/api/v1beta1
+  version: v1beta1
+version: "3"
+```
+
+Replace `path: tutorial.kubebuilder.io/project/api/v1beta1` for 
+`path: tutorial.kubebuilder.io/project/api/batch/v1beta1`
+
+
+In this way, if the project is not new and has previous APIs already implemented will be in the previous structure. 
+Notice that with the `multi-group` project the Kind API's files are
+created under `api/<group>/<version>` instead of `api/<version>`. 
+Also, note that the controllers will be created under `internal/controller/<group>` instead of `internal/controller`. 
+That is the reason why we moved the previously generated APIs with the provided scripts in the previous steps. 
+Remember to update the references afterwards.
```

Comment:
```suggestion
In this process, if the project is not new and has previous implemented APIs they would still need to be modified as needed. 
Notice that with the `multi-group` project the Kind API's files are created under `api/<group>/<version>` instead of `api/<version>`. 
Also, note that the controllers will be created under `internal/controller/<group>` instead of `internal/controller`.
 
That is the reason why we moved the previously generated APIs to their respective locations in the new structure. 
Remember to update the references in imports accordingly.
```